tools/rna_tools/rnaz/rnazCluster.xml

```diff
@@ -0,0 +1,59 @@
+<tool id="rnazCluster" name="rnazCluster.pl" version="0.1.0">
```

Comment:
please add a proper name and description.

Reply:
done